resources_pages/install_ds_stack_mac.md

```diff
@@ -38,6 +38,13 @@ These instructions will walk you through installing the required Data Science so
 
 ## Installation notes
 
+If you have already installed Git, Latex, or any of the R or Python related packages
+please uninstall these and follow the instructions below to reinstall them
+(make sure to also remove any user configuration files and backup them if desired).
+In order to be able to support you effectively
+and minimize setup issues and software conflicts,
+we require all students to install the software stack the same way.
```

Comment:
@ttimbers is this ok or too stringent? I was helping a TA out yesterday who had issues due to previous installations and it takes quite some time to understand what is wrong on a system with software installed in other ways than what we recommend.

I think the programs I mention are the only ones that would be problematic. E.g. I don't think it is a problem if people have VScode installed since earlier and I don't want to tell students to just nuke everything since it can be quite time-consuming with getting back user settings in their browsers etc.